### ISSUE TITLE
Append / to css_base_dir-parameter if it is missing

### ISSUE THREAD

**Issue by Schepp** [TurbineCSS/Turbine#103]
That would be more fool proof.

**Comment by K14D** [comment]
```
---
 lib/base.php |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/lib/base.php b/lib/base.php
index d5556b3..4d485f9 100644
--- a/lib/base.php
+++ b/lib/base.php
@@ -64,9 +64,10 @@ public function __construct(){
    // Try to load config
    @include('config.php');
    if(isset($config)){
-       foreach($config as $key => $setting){
-           $this->config[$key] = $setting;
+       if($config['css_base_dir'] != ''){
+           $config['css_base_dir'] = rtrim($config['css_base_dir'], '/') . '/';
        }
+       $this->config = $config;
    }
    // Set error output
    if($this->config['debug_level'] == 2){
-- 
```

**Comment by Schepp** [comment]
Exactly like this!

**Comment by SirPepe** [comment]
Fixed in f667c28ef5e400299af0288930bf62478bebc493, will be in 1.1.x
